### PR TITLE
IOS-10712: Added left image

### PR DESCRIPTION
**Pull request by WanaldinoTelefonica**
## 🎟️ **Jira ticket**
[IOS-10712](https://jira.tid.es/browse/IOS-10712)

## 🥅 **What's the goal?**
- Add the left image to the Button component

## 🚧 **How do we do it?**
- Follow the same rules as the right image

## 🏑 **AppCenter build**
https://install.appcenter.ms/orgs/Tuenti-Organization/apps/Mistica-SwiftUI-iOS/distribution_groups/public/releases/72
image

### CODE REVIEW

**Comment by DavidMarinCalleja** [comment]
don't we have new screensot testing updated for this component?

**Comment by aweell** [comment]
I downloaded [version 33.2.0(1) from Nov 08, 2024 at 10:29](https://install.appcenter.ms/orgs/Tuenti-Organization/apps/Mistica-SwiftUI-iOS/distribution_groups/public/releases/71) but i can't see what0s displayed in this screenshot:

image

I'm downloading the wrong version?

**Comment by WanaldinoTelefonica** [comment]
> don't we have new screensot testing updated for this component?

Already added new tests for the feature

**Comment by WanaldinoTelefonica** [comment]
> I downloaded [version 33.2.0(1) from Nov 08, 2024 at 10:29](https://install.appcenter.ms/orgs/Tuenti-Organization/apps/Mistica-SwiftUI-iOS/distribution_groups/public/releases/71) but i can't see what's displayed in this screenshot
> 
> I'm downloading the wrong version?
> 
> image

I've regenerated the build in case it was wrong

**Review by alejandroruizponce — APPROVED** [review]

File: Tests/MisticaTests/UI/__Snapshots__/ButtonTests/testLeftImageAndChevron.with-blau-dark-style.png
Comment (yceballost) [inline Tests/MisticaTests/UI/__Snapshots__/ButtonTests/testLeftImageAndChevron.with-blau-dark-style.png:1]: buttonPrimary and secondary dont allow chevron on the right side. This case do not exists in specs

this example in catalog should not be there
image

**Review by yceballost — APPROVED** [review]

**Pull request merged**